### DISCUSSION TITLE
Similar distribution only for boats

### DISCUSSION THREAD

**Discussion by mgrouch** [opencardev/crankshaft#595] [Ideas]
I've built a similar distribution only for boats.

https://github.com/bareboat-necessities/lysmarine_gen

I think some of the things implemented in it could be interesting for auto world.
Touchscreen works. Integration of audio players might be also of interest for crankshaft devs.

Feel free to download and play around.
https://github.com/bareboat-necessities/lysmarine_gen/releases

Thanks,
--MG

**Comment by matt2005** [comment]
Do you have any screenshots or video walkthroughs?

**Reply by mgrouch** [reply]
> Do you have any screenshots or video walkthroughs?

Yes. The links are on project page https://github.com/bareboat-necessities/lysmarine_gen
